File: Question1/conveyor_belt.js

function conveyorBelt (item){
    let conveyorlist = ["bag","shoes"]
    conveyorlist.push(item)
     

    if (conveyorlist.length > 0){
        dequedItem = conveyorlist.shift()

    }
    return dequedItem
    
};

console.log(conveyorBelt("scarf"));

File: Question1/conveyor_belt.py
def queue (item):
     conveyor_list = ["bag","shoes"] 
     conveyor_list.append(item)
     if len(conveyor_list)>0:
         dequeued_item=conveyor_list.pop(0)
         print(dequeued_item)
         
         
         
queue("Coat")
